examples/medplum-demo-bots/package.json

```diff
@@ -44,6 +45,7 @@
     "pdfmake": "0.2.8",
     "ssh2-sftp-client": "9.1.0",
     "stripe": "14.7.0",
+    "tsx": "^4.7.0",
```

Comment:
Was left in by mistake?  `tsx` has a lot of ripple considerations.  it doesn't look like you're using it anywhere.  Let's remove it, and revert changes to `package-lock.json`

`tsx` has some nice properties to it, and I'm open to considering it, but if we're going to use it, we should discuss at large

Reply:
I don't use it anywhere in the code, but just to be able to run the script. I'm fine to remove it and do a global install of `tsx` on my machine instead. 

I didn't really want to use it. my goal was to use
`npx ts-node ....`
But I was running into [this bug](https://github.com/TypeStrong/ts-node/issues/1997)  when using `ts-node`

Reply:
Sigh, yeah

Here's what I do for my one-off scripts:

```bash
node --no-warnings=ExperimentalWarning --loader ts-node/esm src/index.ts
```

Tragically complicated.

(*cough* wouldn't be a problem with Bun *cough*)